zanata-war/src/main/java/org/zanata/action/PasswordResetAction.java

```diff
@@ -79,27 +84,24 @@ public boolean validatePasswordsMatch() {
 
     public void setActivationKey(String activationKey) {
         this.activationKey = activationKey;
-        key =
-                entityManager.find(HAccountResetPasswordKey.class,
+        key = entityManager.find(HAccountResetPasswordKey.class,
                         getActivationKey());
     }
 
-    // @Begin(join = true) /* TODO [CDI] commented out begin conversation. Verify it still works properly */
+    // @Begin(join = true)
```

Comment:
Given the trouble we've had with scopes, I can't rule out the possibility
of switching some beans to conversation-based scopes at some point, so I'd
prefer to keep the hints about where we were starting and ending the
conversations in Seam.

On 1 March 2016 at 09:44, Carlos Munoz <notifications@github.com> wrote:

> In zanata-war/src/main/java/org/zanata/action/PasswordResetAction.java
> <https://github.com/zanata/zanata-server/pull/1072#discussion_r54500034>:
>
> >                          getActivationKey());
> >      }
> >
> > -    // @Begin(join = true) /* TODO [CDI] commented out begin conversation. Verify it still works properly */
> > +    // @Begin(join = true)
>
> another unnecessary comment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1072/files#r54500034>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat